### DISCUSSION TITLE
set p-menu-item-link-active

### DISCUSSION THREAD

**Discussion by foum-traore** [primefaces/community#2748] [PrimeNG]
hello,
I'm using 18.0.0-beta.2
how to set p-menu-item-link-active background color.

the css color variables still have the same name in v18?
 .p-menu-item-link-active {
    background-color: var(--primary-color);
    color: white;
  }

**Comment by qwadrox** [comment]
Hi @foum-traore. As I see the source code, `p-menu-item-link-active` is the name of the class used in the simple menu component: https://github.com/primefaces/primeng/blob/3b4d78268e25eed3238d892f2ec36475c159c9e9/src/app/components/menu/menu.ts#L91
So [the docs](https://v18.primeng.org/menu#router) is not correct mentioning `p-menuitem-link-active`
I don't fully understand what you mean with the CSS color variable. 
You can find the theming tokens of the menu on the theming tab: 
<img width="492" alt="image" src="https://github.com/user-attachments/assets/ca443c5b-5c66-408a-8888-141db3022325">

18 is still very beta, so if you're not confident searching in the source code, I would recommend to stick to the stable version.
